app/src/main/java/io/islnd/android/islnd/app/database/IslndDb.java

```diff
@@ -268,4 +270,33 @@ public static void deleteComment(
         // Delete from network
         Rest.postComment(encryptedComment, Util.getApiKey(context));
     }
+
+    public static void syncServerTime(Context context, boolean force) {
+        final String prefKey = context.getString(R.string.server_time_offset);
+
+        SharedPreferences settings = PreferenceManager.getDefaultSharedPreferences(context);
+        String savedOffset = settings.getString(prefKey, null);
+
+        if (force || savedOffset == null) {
+            // get from server
+            new AsyncTask<Void, Void, Void>() {
+                @Override
+                protected Void doInBackground(Void... params) {
+                    try {
+                        long serverTimeOffset = ServerTime.synchronize(context);
+                        settings.edit()
+                                .putString(prefKey, Long.toString(serverTimeOffset))
+                                .apply();
```

Comment:
This is cool. I didn't know about the apply method, but it looks like it is what we should be using most of the time. Good catch :+1